src/pages/index.js

```diff
@@ -2,75 +2,29 @@ import {nanoid} from 'nanoid';
 import {useState} from 'react';
 import {Helmet} from 'react-helmet';
 
-import Button from '../components/Button';
 import Layout from '../components/Layout';
-import useFetch from '../hooks/useFetch';
-import useStore from '../hooks/useStore';
+import QuestionComponent from '../components/QuestionComponent';
 
 export default function HomePage() {
-	// Data
-	const {data, loading, error} = useFetch('/api/hello');
-
-	// Local state
-	const [id, setId] = useState(null);
-
-	// Global state ZUSTAND
-	const counter = useStore(state => state.counter);
-	const decrementCounter = useStore(state => state.decrementCounter);
-	const incrementCounter = useStore(state => state.incrementCounter);
-	const setCounter = useStore(state => state.setCounter);
+	const [questionArray, setQuestionArray] = useState([
+		{questionText: 'What is export default?'},
+		{questionText: 'What does clean code means?'},
+		{questionText: 'Why do we have to define keys?'},
+	]);
 
 	return (
 		<Layout>
 			<Helmet>
 				<title key="title">My Project</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Helmet>
-			<h1>Home</h1>
-			{loading && <div>Loading...</div>}
-			{error && <div>{error.message}</div>}
-			{data && (
-				<pre>
-					<code>{JSON.stringify(data, null, 4)}</code>
-				</pre>
-			)}
-			<section>
-				<Button
-					aria-label="decrement"
-					onClick={() => {
-						decrementCounter();
-					}}
-				>
-					-
-				</Button>
-				<input
-					value={`${counter}`}
-					size={2}
-					onChange={event => {
-						setCounter(Number.parseInt(event.target.value, 10));
-					}}
-				/>
-				<Button
-					aria-label="increment"
-					onClick={() => {
-						incrementCounter();
-					}}
-				>
-					+
-				</Button>
-			</section>
-			<br />
-			<section>
-				<Button
-					onClick={() => {
-						setId(nanoid());
-					}}
-				>
-					Generate ID
-				</Button>
-				<br />
-				<div>Id: {id}</div>
-			</section>
+			<article>
```

Comment:
I'm not sure if you really need an extra `article` element. 🤔 Your questions will be rendered inside of a `main` element and for me this would be semantic enough.

Reply:
You are totally right. I just thought one step ahead again. :)
But this article will only be necessary in the next steps! 
I removed it for now